### ISSUE TITLE
Allow defining individual bar height

### ISSUE THREAD

**Issue by kaustubhhiware** [dpayne/cli-visualizer#66]
Is it possible to set maximum bar height some way ?
The above terminal in tilix, where ~16 colour boxes are visible, whereas in the lower xfce-terminal only 6 are visible, due to which the red bar's height is _waay_ more than the rest blocks in the lower terminal.

I wanted to get the output of the above terminal in the lower terminal as well (number of blocks). How can I do that ?

![vis_load](https://user-images.githubusercontent.com/11318551/31066293-6e498f28-a769-11e7-85ee-a5c592bdda72.png)

[config](https://gist.github.com/kaustubhhiware/deaea84b53f5de13e7be39dffb84b769)

**Comment by dpayne** [comment]
I think this is just a bug when there are less colors than bar height. I'll work on a fix. However, is there a reason why you want less than 16 colors? Your terminal _should_ support 256 colors. Could you try setting `export TERM=rxvt-256color`?

**Comment by kaustubhhiware** [comment]
> is there a reason why you want less than 16 colors? 
Actually I prefer the 256 color version, I've understood what the problem could be. I'll find a fix (if it's related to xfce-terminal) and let you know soon.

**Comment by kaustubhhiware** [comment]
I finally managed to get it working. Since the lower terminal was xfce-terminal, it required me to run the command while setting the environment variable `TERM` as `xterm-256color`

While adding this to ~/.bashrc didn't work, it is possible to change a variable for just one command.

Command needed to run: `env TERM=xterm-256color vis`

The outcome of this command now is exactly what I wanted.

![vis_load](https://user-images.githubusercontent.com/11318551/31114569-605ced50-a83c-11e7-8361-ef2ac644377a.png)

Closing the issue now. Thanks for your time.
